package.json

```diff
@@ -58,17 +60,17 @@
     "popper.js": "1.16.1",
     "process": "0.11.10",
     "prop-types": "15.8.1",
-    "react": "16.13.1",
-    "react-beautiful-dnd": "13.1.1",
+    "react": "17.0.2",
+    "react-beautiful-dnd": "13.1.0",
```

Comment:
Did you intend to downgrade this one?

Reply:
This was related to previous react 17 pr. Probably wont be included in actual pr.